src/compiler/app-core/app-data.ts

```diff
@@ -129,7 +137,15 @@ const getModuleImports = (moduleMap: ModuleMap, filePath: string, importedModule
   return importedModules;
 };
 
-export const updateBuildConditionals = (config: Config, b: BuildConditionals) => {
+/**
+ * For a provided Stencil project configuration, update provided the build conditionals
```

Comment:
```suggestion
 * Update the provided build conditionals object in-line with a provided Stencil project configuration
```

Reply:
Sometimes I really wish TypeScript had a `mut` keyword that could be used to signal this (and to produce a type error if you try to mutate something without it). Ah well!

Reply:
Done in cafda3a87

> Sometimes I really wish TypeScript had a mut keyword that could be used to signal this (and to produce a type error if you try to mutate something without it). Ah well!

Oh, I'd love somethings like that! Terrified what edge cases _implementing_ something like that might look like though 😬 😆